content/chainguard/libraries/overview.md

```diff
@@ -59,9 +59,10 @@ contents into our production environment.
 ## Introduction 
 
 Chainguard Libraries builds all available libraries from source code in the
-Chainguard Factory and makes them available for you. Chainguard Factory is a
-more secure, dedicated, private, and SLSA-certified build infrastructure for
-building software from source and publishing the binaries to customers.
+Chainguard Factory and makes them available for you. Chainguard Factory
+represents Chainguard's internal tooling that enable a more secure, dedicated,
```

Comment:
Consider "is what we call" in place of "represents", but this is just a thought, not a demand.

Reply:
Lisa suggested that and I also think "represents" sounds more professional.. not sure yet about the overall voice of our docs just yet so I am going to leave as is